implementation-report.md

```diff
@@ -1,3 +1,10 @@
 # WebDriver implementation report
 
-[to come]
+Currently tests are being tracked on [http://wpt.fyi/webdriver](http://wpt.fyi/webdriver).
+
+## Status Documents from Vendors
+
+* [Mozilla Firefox](https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver/status)
+* [Microsoft Edge](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webdriver/)
+* [Apple Safari](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webdriver/)
+* WebKit (Implementation Report to be added)
```

Comment:
The WebKit GTK port, specifically. Safari also uses WebKit, so they share the same interfaces but have different implementations.